src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calc from './components/calculator';
 
 function App() {
```

Comment:
The App.js component should also be class-based.

Reply:
Hi dear @keneogu,
Thank you for reviewing my code.
Can you show me the exact bottom margin of - + and x inside the code? because I can't find the bottom margin in my style. Actually, throughout the whole project, I didn't use 'margin' or 'margin-bottom'.
Thank you again.